example/src/SiteWrapper.react.js

```diff
@@ -160,8 +164,14 @@ const accountDropdownProps = {
   ],
 };
 
-class SiteWrapper extends React.Component<Props, void> {
+class SiteWrapper extends React.Component<Props, State> {
+  constructor() {
```

Comment:
This can be simplified to the shorthand notation we have been using:

```
state = {
    unreadCount: 2
};
```

Reply:
Sorry about that, I am not a typescript developer (I used to be, long story).